class ExceptionHandler:
    def log_exception(self):
        pass
